tests/external-signed-ca-with-manual-copy/install-server-with-external-ca-with-manual-copy.yml

```diff
@@ -10,15 +10,14 @@
     state: present
 
   post_tasks:
-  - name: Copy CSR /root/ipa.csr from node to "{{ groups.ipaserver[0] + '-ipa.csr' }}"
+  - name: Copy CSR /root/ipa.csr from node to "{{ ansible_fqdn + '-ipa.csr' }}"
```

Comment:
Also here

Reply:
Are you sure? We have `hosts: ipaserver` in this playbook.